src/ModalForms.js

```diff
@@ -0,0 +1,59 @@
+import React, { Fragment } from 'react';
+import Modal from './Modal';
+
+class InfoModal extends React.Component {
+  componentWillMount() {
+    this.setState({ showModal: false });
```

Comment:
Once again this needs to be in the parent and you need to pass a prop with a method that changes that.

Reply:
See comment here: https://github.com/TechForUK/simple-vote-ui/pull/54#discussion_r280982760